src/components/ProposalIcon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { GU } from '@1hive/1hive-ui'
+import { ProposalTypes } from '../types'
+
+function ProposalIcon({ type }) {
+  let color
+  if (type === ProposalTypes.Decision) {
+    color = '#FFC3AB'
```

Comment:
we don't have those colors on the ui right?

Reply:
Ah actually we do have the Proposal and Suggestion colors, will use them 👍 
Though we don't have the one for the Decision